server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/GetFieldMappingsIndexRequest.java

```diff
@@ -45,12 +44,13 @@
         }
         fields = in.readStringArray();
         includeDefaults = in.readBoolean();
-        probablySingleFieldRequest = in.readBoolean();
+        if (in.getVersion().before(Version.V_8_0_0)) {
```

Comment:
Small comment, we could add a comment here explaining this backwards-compatibility logic.